src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Todos{
+
+    protected String type = "E";
+    Event(String name, boolean marked, String type) {
+        super(name, marked, type);
+    }
+    public String getType() {
+        return "[E]";
+    }
+
```

Comment:
Not code quality, but to follow conventions, I recommend removing all these extra empty lines.